x/gov/genesis_test.go

```diff
@@ -111,6 +113,27 @@ func TestImportExportQueues(t *testing.T) {
 	require.True(t, proposal2.Status == types.StatusRejected)
 }
 
+func TestImportExportQueues_ErrorUnconsistentState(t *testing.T) {
+	app := simapp.Setup(false)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
+	require.Panics(t, func() {
+		gov.InitGenesis(ctx, app.AccountKeeper, app.BankKeeper, app.GovKeeper, &types.GenesisState{
+			Deposits: types.Deposits{
+				{
+					1234,
+					"me",
+					sdk.Coins{
+						{
+							"stake",
+							sdk.NewInt(1234),
+						},
+					},
+				},
+			},
+		})
+	})
```

Comment:
could you add a comment? What's wrong here ? why this test should panic?

Reply:
it tests genesis panics when there's inconsistent state, this change is commented in the changelog and in the places where the panic happens (i've also addressed why this change is needed above in the comments)